.github/workflows/main.yml

```
@@ -44,7 +44,7 @@ jobs:
     secrets: inherit
     with:
       CONTEXT: dev
-      ROLE_TO_ASSUME: arn:aws:iam::634210816736:role/sagebase-github-oidc-sage-bionetworks-dccvalidator-1kD-infra
+      ROLE_TO_ASSUME: arn:aws:iam::634210816736:role/sagebase-github-oidc-dccvalidator-deploy
```

```suggestion
      ROLE_TO_ASSUME: arn:aws:iam::634210816736:role/sagebase-github-oidc-dcc-validator-deploy
```

It looks like the dependent PR is creating `dcc-validator-deploy`.

---

nice catch.  fixed by changing it to `dccvalidator-deploy` in the dependent PR